endpoints.json

```diff
@@ -25,5 +25,23 @@
         }
       ]
     }
+  },
+  "GET /api/articles/:article_id": {
+    "description": "serves an object containing one article",
+    "queries": [],
+    "exampleResponse": {
+      "articles": [
```

Comment:
Please dont copy and paste. This should be "article" not "articles" and it should have an object value not an array.